examples/pettingzoo/utils.py

```diff
@@ -91,8 +91,8 @@ def step(self, action):
     if done:
       self.agents = []
 
-    observations = utils.timestep_to_observations(timestep)
-    return observations, rewards, dones, infos
+    observations, _ = utils.timestep_to_observations(timestep)
+    return observations, rewards, dones, dones, infos
```

Comment:
Does the first dones need __all__?
https://docs.ray.io/en/latest/_modules/ray/rllib/env/multi_agent_env.html#make_multi_agent

Reply:
OK lets see how it goes :-)